foldamers/cg_model/cgmodel.py

```diff
@@ -570,7 +673,7 @@ def get_bond_angle_list(self):
                         bond_angles.append(bond_angle)
         return bond_angles
 
-    def get_torsion_list(self):
+    def get_torsion_list(self):  # MRS: really slow, should be looked at.
```

Comment:
I have some ideas for making this more efficient - I can make a separate issue.

I usually find torsions by first computing the number of bonds each bead is a part of, then only looking at bonds in which each atom has connectivity >= 2 as the torsion centers.

Reply:
Probably can be a separate issue, it's not THAT slow (a few seconds for a short heteropolymer, longer than it should give the number of torsions is not that large). But it appears to work correctly, which is the most important.